drv/Iterators.drv

```diff
@@ -877,7 +877,10 @@ public final class ITERATORS {
 
 		@Deprecated
 		@Override
-		public KEY_GENERIC_CLASS next() { return KEY_GENERIC_CLASS.valueOf(iterator.nextByte()); }
+		public KEY_GENERIC_CLASS next() {
+			assert false : "Use the primitive next..() method instead.";
```

Comment:
nextByte() same for others

Reply:
the actual function is type dependent, for instance, for IntIterators, it will be nextInt(), not nextByte().